collectors/timex.plugin/README.md

```diff
@@ -0,0 +1,25 @@
+<!--
+title: "timex.plugin"
+description: "Monitor the system clock synchronization state."
+custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/timex.plugin/README.md
+-->
+
+# timex.plugin
+
+This plugin monitors the system clock synchronization state, under Linux.
+
+Two charts are available:
+
+-   System Clock Synchronization State
+-   Computed Time Offset Between Local System and Reference Clock
+
+## configuration
+
```

Comment:
It would be great if we could add our little bit of boilerplate here about configuration.

```
Edit the `netdata.conf` configuration file using [`edit-config`](/docs/configure/nodes.md#use-edit-config-to-edit-configuration-files) from the [Netdata config directory](/docs/configure/nodes.md#the-netdata-config-directory), which is typically at `/etc/netdata`.

Scroll down to the `[plugin:timex]` section to find the available options:
```

Reply:
Done.